src/templates/actionBar.jsx

```diff
@@ -136,185 +164,217 @@ class ActionBar extends React.PureComponent {
-                                                            !iconFilter &&
-                                                            !iconGrid &&
-                                                            !iconSearch &&
-                                                            !iconTable
-                                                    ) {
-                                                        console.warn( // eslint-disable-line no-console, max-len
-                                                            `${'<Page.ActionsBar>\'s column.list object must have one of the ' +
-                                                                'following properties: actionsButton, iconBack, iconFilter, iconGrid, or iconTable.' +
-                                                                'Please check this column.list object '}${JSON.stringify(item)}`,
-                                                        );
-
-                                                        return false;
-                                                    }
-
-                                                    if (iconSearch) {
-                                                        /* eslint-disable max-len */
-                                                        /* eslint-disable react/jsx-props-no-spreading */
-                                                        searchDataForMobile = (
-                                                            <ActionBarSearch
-                                                                {...iconSearch}
-                                                                isMobileSearch
-                                                                isMobileSearchVisible={isMobileSearchVisible}
-                                                                type="mobileSearch"
+                                            {dropdownButton && (
+                                                <DropdownButton
+                                                    className={dropdownButton.className}
+                                                    color={dropdownButton.color}
+                                                    disable={dropdownButton.disable}
+                                                    iconType={dropdownButton.iconType}
+                                                    id={dropdownButton.id}
+                                                    label={dropdownButton.label}
+                                                    onClick={dropdownButton.onClick}
+                                                    style={dropdownButton.style}
+                                                    title={dropdownButton.title}
+                                                >
+                                                    {_.map(dropdownButton.options, (option) => {
+                                                        dropdownButtonOptionKeyNum += 1;
```

Comment:
Look at you not using unary `++` and then suppressing the warning! 😁 
I think I'm perhaps a bit more of a rebel...